queue/testinit.go

```diff
@@ -5,7 +5,7 @@ var IsTest = false
 func SetIsTest() {
 	IsTest = true
 
-	Init()
+	Start()
 }
 
 func UnsetIsTest() {
```

Comment:
Found that there is some historical issue in the test,
in that we may want to Close in the UnsetIsTest for memory cleaning.
(SetIsTest and UnsetIsTest are used in api/testinit_test.go and cron/testinit_test.go)
(The purpose of SetIsTest is to let the module know that it is in test-mode.)
(Basically it's supposed to be with only 1 test-function at a time (in serial),
 or the data in mongodb / redis will be contaminated among tests)

The reason why we still pass the tests is because the number of the queue is small (originally it was only 2),
but I'm kind of worried that the tests may fail if we have more tests.